### PR TITLE
[php2cpg] Fixed the initialization of static/const members of class

### PR DESCRIPTION
**Pull request by d1tto**
1. The initialization of static members of class should be placed in `<clinit>` function, not in the constructor of the class. 
2. Access to static members should be prefixed with `self` keyword to distinguish them from normal class members. For example:
    ```php
    <?php
    class Foo {
        static $A = "A";
        const B = "B";
        function foo() {
            echo self::$A;
            echo self::B;
        }
    }
    ```

### CODE REVIEW

**Comment by d1tto** [comment]
@DavidBakerEffendi Excuse me, could you help me review this pr

File: joern-cli/frontends/php2cpg/src/main/scala/io/joern/php2cpg/astcreation/AstCreator.scala
Comment (DavidBakerEffendi) [inline joern-cli/frontends/php2cpg/src/main/scala/io/joern/php2cpg/astcreation/AstCreator.scala:891]: from `io.joern.x2cpg.Defines`
```suggestion
        newIdentifierNode(name, typ.getOrElse(Defines.Any), typ.toList, memberNode.lineNumber).code(name)
```

File: joern-cli/frontends/php2cpg/src/main/scala/io/joern/php2cpg/astcreation/AstCreator.scala
Comment (DavidBakerEffendi) [inline joern-cli/frontends/php2cpg/src/main/scala/io/joern/php2cpg/astcreation/AstCreator.scala:894]: I think should work, and if it does will be more concise
```suggestion
      val code = s"self::${memberNode.code.replaceAll("(static|case|const) ", "")}"
```

**Review by DavidBakerEffendi — COMMENTED** [review]
I'll leave the final say with @johannescoetzee

File: joern-cli/frontends/php2cpg/src/main/scala/io/joern/php2cpg/astcreation/AstCreator.scala
Comment (d1tto) [reply joern-cli/frontends/php2cpg/src/main/scala/io/joern/php2cpg/astcreation/AstCreator.scala]: Applied. Thanks

**Comment by d1tto** [comment]
Excuse me, is there any progress on this pr 😛

**Review by DavidBakerEffendi — APPROVED** [review]
I'm happy to merge this, then if it breaks something for stakeholders we can always revert worst case

**Pull request merged**
